packages/fast-color-explorer/app/app.tsx

```diff
@@ -27,6 +28,7 @@ import {
     ColorRecipe,
     Swatch,
 } from "@microsoft/fast-components-styles-msft/dist/utilities/color/common";
```

Comment:
Are we not exporting that enum for a reason?

Reply:
I ran bootstrap again and now it works.